Tools/autotest/vehicle_test_suite.py

```diff
@@ -2559,64 +2559,51 @@ def test_adsb_send_threatening_adsb_message(self, here, offset_ne=None):
-            "SIM_BAR3_WCF_FWD",
-            "SIM_BAR3_WCF_LFT",
-            "SIM_BAR3_WCF_RGT",
-            "SIM_BAR3_WCF_UP",
-            "SIM_BARO_COUNT",
-            "SIM_BARO_DELAY",
-            "SIM_BARO_DISABLE",
-            "SIM_BARO_FREEZE",
-            "SIM_BARO_WCF_BAK",
-            "SIM_BARO_WCF_DN",
-            "SIM_BARO_WCF_FWD",
-            "SIM_BARO_WCF_LFT",
-            "SIM_BARO_WCF_RGT",
-            "SIM_BARO_WCF_UP",
-            "SIM_BATT_CAP_AH",
-            "SIM_BAUDLIMIT_EN",
-            "SIM_DRIFT_SPEED",
-            "SIM_DRIFT_TIME",
-            "SIM_EFI_TYPE",
-            "SIM_ESC_ARM_RPM",
-            "SIM_FTOWESC_ENA",
-            "SIM_FTOWESC_POW",
+            "SIM_ACC_TRIM_X",    # not listed in SITL.cpp
+            "SIM_ACC_TRIM_Y",    # not listed in SITL.cpp
+            "SIM_ACC_TRIM_Z",    # not listed in SITL.cpp
+            "SIM_ARSPD2_OFS",    # not listed in SITL.cpp
+            "SIM_ARSPD2_RND",    # not referenced anywhere
+            "SIM_ARSPD_OFS",     # not referenced anywhere
+            "SIM_ARSPD_RND",     # not listed in SITL.cpp
+            "SIM_BAR2_DELAY",    # not referenced anywhere
+            "SIM_BAR2_DISABLE",  # not listed in SITL.cpp
+            "SIM_BAR2_DRIFT",    # not referenced anywhere
+            "SIM_BAR2_FREEZE",   # not listed in SITL.cpp
+            "SIM_BAR2_WCF_BAK",  # not referenced anywhere
+            "SIM_BAR2_WCF_DN",   # not referenced anywhere
+            "SIM_BAR2_WCF_FWD",  # not referenced anywhere
+            "SIM_BAR2_WCF_LFT",  # not referenced anywhere
+            "SIM_BAR2_WCF_RGT",  # not referenced anywhere
+            "SIM_BAR2_WCF_UP",   # not referenced anywhere
+            "SIM_BAR3_DELAY",    # not referenced anywhere
+            "SIM_BAR3_DISABLE",  # not referenced anywhere
+            "SIM_BAR3_DRIFT",    # not referenced anywhere
+            "SIM_BAR3_FREEZE",   # not referenced anywhere
+            "SIM_BAR3_WCF_BAK",  # not referenced anywhere
+            "SIM_BAR3_WCF_DN",   # not referenced anywhere
+            "SIM_BAR3_WCF_FWD",  # not referenced anywhere
+            "SIM_BAR3_WCF_LFT",  # not referenced anywhere
+            "SIM_BAR3_WCF_RGT",  # not referenced anywhere
+            "SIM_BAR3_WCF_UP",   # not referenced anywhere
+            "SIM_BARO_DELAY",    # not listed in SITL.cpp
+            "SIM_BARO_DISABLE",  # not listed in SITL.cpp
+            "SIM_BARO_FREEZE",   # not referenced anywhere
+            "SIM_BARO_WCF_BAK",  # not listed in SITL.cpp
+            "SIM_BARO_WCF_DN",   # not listed in SITL.cpp
+            "SIM_BARO_WCF_FWD",  # not listed in SITL.cpp
+            "SIM_BARO_WCF_LFT",  # not listed in SITL.cpp
+            "SIM_BARO_WCF_RGT",  # not listed in SITL.cpp
+            "SIM_BARO_WCF_UP",   # not listed in SITL.cpp
+            "SIM_FTOWESC_ENA",   # not listed in SITL.cpp
+            "SIM_FTOWESC_POW",   # not listed in SITL.cpp
```

Comment:
These will be in SIM_FETTechOneWireESC

Reply:
I see, I have removed those param comments